src/state/queries/suggested-feeds.ts

```diff
@@ -0,0 +1,29 @@
+import {useInfiniteQuery, InfiniteData, QueryKey} from '@tanstack/react-query'
+import {AppBskyFeedGetSuggestedFeeds} from '@atproto/api'
+
+import {useSession} from '#/state/session'
+
+export const suggestedFeedsQueryKey = ['suggestedFeeds']
+
+export function useSuggestedFeedsQuery() {
+  const {agent} = useSession()
+
+  return useInfiniteQuery<
+    AppBskyFeedGetSuggestedFeeds.OutputSchema,
+    Error,
+    InfiniteData<AppBskyFeedGetSuggestedFeeds.OutputSchema>,
+    QueryKey,
+    string | undefined
+  >({
+    queryKey: suggestedFeedsQueryKey,
+    queryFn: async ({pageParam}) => {
+      const res = await agent.app.bsky.feed.getSuggestedFeeds({
+        limit: 10,
+        cursor: pageParam,
+      })
```

Comment:
We only return 10 right now regardless, but in the future we may actually paginate